addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -293,20 +293,15 @@ def send_pec_mail(self):
                 )
             self.l10n_it_send_state = 'invalid'
 
-    def _import_xml_invoice(self, content, attachment):
+    def _import_xml_invoice(self, tree):
```

Comment:
there is also the _create_invoice_from_mail method in l10n_it_edi/models/ir_mail_server.py that calls this method and since you've changed the definition of the method, you have to adapt it there also.